R/geo.R

```diff
@@ -167,4 +167,178 @@ ward_enr_aggs <- function(df) {
 }
 
 
+#' Aggregates assessment data by ward
+#'
+#'
+#' @param list_of_dfs output of \code{fetch_all_parcc}, \code{fetch_parcc},
+#' \code{calculate_agg_parcc_prof}
+#'
+#' @return A data frame of ward aggregations
+#'
+#'
+#' @export
+ward_parcc_aggs <- function(list_of_dfs) {
+   
+   df <- list_of_dfs %>%
+      bind_rows() %>% # convert to df
+      enrich_school_latlong() %>%
+      enrich_school_city_ward()
+   
+   df <- df %>%
+      filter(!is.na(ward) #,
+             #subgroup == "special_education",
```

Comment:
probably had these in here for testing / dev purposes?  would be good to clean these up before merging the final version

Reply:
oops -- fixed adb4377396b89189e93b10ae405f31bc36ba9d0f